templates/named.conf.options.erb

```diff
@@ -22,9 +22,9 @@ options {
 <% end -%>
 
 <% if @listen_on.size == 0 then -%>
```

Comment:
oh, I just looked at the [official docs](https://kb.isc.org/article/AA-01031) and there actually is a `port` option, separate from the `listen-on` option.  I'll update the template to use that instead.